flaxandteal/coral-arches
# Fix(#2316): license function errors when importing

**Description**

Problems during licence resource imports have been resolved. No more cardinality errors and validation of the status tile existing re-creating itself.

There is still the problem of the License Name Nodegroup tile and License Number Nodegroup which will need to be removed. The cardinality issues not related to these is what has been fixed.

**Tests**
1. Create a new licence workflow
1. Complete the first step
1. Export the business data for it and rename the file to license_without_number
1. Delete all instances for licence
1. Create a new licence workflow
1. Complete the first step
1. On application details mark it as granted for the status
1. Navigate to record decision
1. Mark it approved from cur d
1. A license number should have generated
1. Export the business data for it and rename the file to license_with_number
1. Delete all instances for licence
1. Attempt to import license_without_number
1. It will fail with cardinality issues
1. Delete the tile on license_without_number with this nodegroup id 59d65ec0-48b9-11ee-84da-0242ac140007 (License Name)
1. Attempt to import license_without_number
1. It will fail with cardinality issues
1. Delete the tile on license_without_number with this nodegroup id 6de3741e-c502-11ee-86cf-0242ac180006 (License Number)
1. Attempt to import license_without_number
1. It will pass
1. Repeat steps from 13 except for the license_with_number business data
